subsys/bluetooth/host/monitor.c

```diff
@@ -126,12 +126,13 @@ static void monitor_send(const void *data, size_t len)
 	}
 
 	if (!drop) {
-		if (!panic_mode) {
+		if (panic_mode) {
+			cnt = SEGGER_RTT_WriteNoLock(CONFIG_BT_DEBUG_MONITOR_RTT_BUFFER,
+										 rtt_buf, rtt_buf_offset);
+		} else {
 			SEGGER_RTT_LOCK();
-		}
-		cnt = SEGGER_RTT_WriteNoLock(CONFIG_BT_DEBUG_MONITOR_RTT_BUFFER,
-					     rtt_buf, rtt_buf_offset);
-		if (!panic_mode) {
+			cnt = SEGGER_RTT_WriteNoLock(CONFIG_BT_DEBUG_MONITOR_RTT_BUFFER,
+										 rtt_buf, rtt_buf_offset);
 			SEGGER_RTT_UNLOCK();
```

Comment:
I think `SEGGER_RTT_Write` is shorter way to write this. I suggest using that.

Reply:
Nice, I have not notice that, Thank You!